task 2.py

```diff
@@ -0,0 +1,5 @@
+a = int(input())
+k = int((a/3)*2)
+p = int((k/2)/2)
+s = int(p)
```

Comment:
Во-первых, у операций деления и умножения одинаковый приоритет. Во-вторых, функцию `int` использовать не обязательно. Для этого у нас существуют отличный оператор целочисленного отделения.
```suggestion
k = a // 3 * 2
p = s = k // 2 // 2
```